### ISSUE TITLE
Ban interpolated anime clips from the "For You" feed

### ISSUE THREAD

**Issue by irgendwr** [twitter/the-algorithm#386]
**Is your feature request related to a problem? Please describe.**
Seeing interpolated 60fps anime clips has a negative impact on my mental health

**Describe the solution you'd like**
Videos of interpolated anime clips should either be banned outright

**Describe alternatives you've considered**
Add a feature to block specific media, so that I can block specific videos

**Additional context**
See this tweet for context: https://twitter.com/irgendwr/status/1641608401636536321
(the quoted tweet is an example of media that should be banned)

**Comment by mitsokoyay** [comment]
No, I really like watching my anime in 60fps. I think the original artists were clowns and the world's art supply will be expanded and improved by ChatGTP
